### DISCUSSION TITLE
create an official telegram group on lxqt

### DISCUSSION THREAD

**Discussion by telmobss1** [lxqt/lxqt#1940] [Ideas]
good night guys, i'm just a regular user who doesn't know how to change the code, but i can help with the translation of the language into portuguese portuguese as well as contribute with ideas

recently, i saw that there wasn’t an official lxqt group on the telegram (biggest chat platform) so i would like to ask if there is space for ordinary users to give their ideas and opinions, contribute to the tests, as well as other things,  and the most fundamental part, listening to the community

**Comment by tsujan** [comment]
> ... to give their ideas and opinions, contribute to the tests, as well as other things, and the most fundamental part, listening to the community

Yes, of course! All of them would be very appreciated, especially testing.

**Reply by tsujan** [reply]
P.S.
What I meant was about contributions by ordinary users. I have no opinion about an official telegram group (personally, I don't have time for it). Let's wait to hear from other LXQt members.

**Comment by wxl** [comment]
Right now we have an IRC channel and a Matrix room. It would be nice if we bridged those together (which is pretty trivial to whomever has control of the Matrix room, as they should just be able to add \#oftc_#lxqt:matrix.org as an alias).

I don't really love Telegram, personally, but I can tell you that when we (Lubuntu) added it, we got a lot of additional feedback… and contributors. The thing that made it work is that we bridged it to our main mode of communication (IRC which is essentially already bridged to Matrix as aforementioned). Matterbridge is what we're currently doing for this. I don't know that we have any infrastructure at this point, but if we did, we could run it and we'd be all set.

Without this, it requires contributors to be in the Telegram channel and that may be assuming too much. Even right now, IRC and Matrix discussions are almost nil. Perhaps what could justify all this is someone making a Telegram channel and monitoring it. I could probably come up with some folks that could do that if need be.

**Reply by telmobss1** [reply]
on my part, i can create a group on the telegram considered official, and then over time, if they were interested we would integrate irc with the telegram group 

Do you think it's a good proposal?

**Reply by monetaryabyss** [reply]
If you decide to create a group, please keep us posted. I would be willing to help monitor things, I also help admin the Telegram side of the  Lubuntu channels so I have some experience with that too.

**Reply by stefonarch** [reply]
> It would be nice if we bridged those together (which is pretty trivial to whomever has control of the Matrix room, as they should just be able to add #oftc_#lxqt:matrix.org as an alias).

Matrix room was created by @paulolieuthier  https://github.com/lxqt/lxqt/discussions/1923
Bridging would be really effective in not splitting the discussions.

**Reply by telmobss1** [reply]
thank you so i will create a group as soon as i have time to organize the work

**Reply by telmobss1** [reply]
Good morning, I just created the group and I already have everything ready to integrate the matrix to the telegram, but for that to happen I need the permission of the administrator 


I leave the telegram link below, and the text that some of the matrix administrators have to make so that the connection between telegram and matrix happens


https://t.me/lxqtofficial
the words to be placed in the matrix is ​​this


!tg bridge -506715938

**Reply by telmobss1** [reply]
to connect the telegram to the matrix, I need anyone who is a moderator here to give permission

@paulolieuthier

**Reply by stefonarch** [reply]
Updated https://github.com/lxqt/lxqt/wiki/Infrastructure-Fellows
Old bus factor issue, we need at least 2 people everywhere...

**Reply by telmobss1** [reply]
Its Telmo1241 and not 2141
Just to make it clear

But thanks for adding

**Reply by stefonarch** [reply]
oops sorry ;)

**Reply by paulolieuthier** [reply]
> It would be nice if we bridged those together (which is pretty trivial to whomever has control of the Matrix room, as they should just be able to add #oftc_#lxqt:matrix.org as an alias).

Is that all that's needed? I'm trying to setup integration through the integration settings, and I need an IRC operator live on the channel to confirm it.

**Reply by wxl** [reply]
> > It would be nice if we bridged those together (which is pretty trivial to whomever has control of the Matrix room, as they should just be able to add #oftc_#lxqt:matrix.org as an alias).
> Is that all that's needed? I'm trying to setup integration through the integration settings, and I need an IRC operator live on the channel to confirm it.

That should be it. Unfortunately I don't remember how I set this up for Lubuntu way back when but we definitely have the two bridged and it's not because of our bridge for Telegram (I further confirmed this by looking at our config file for Matterbridge right now). 

To be clear, there is a [bridge service](https://github.com/matrix-org/matrix-appservice-irc) for IRC/Matrix. Anyone can host one, but the canonical instance is on matrix.org (thus the reason why matrix.org is in the room name even though we're trying to connect to IRC). In this sense, Matrix users can (almost) any IRC channel without needing to have any special bridging or anything.

The problem is that we have a Matrix room and an IRC channel separate from one another and we need to put two and two together.

That said, I'm on Matrix and on IRC if you need help @paulolieuthier.

BTW it looks like though we have 7 ops defined for the IRC channel, only two seem to actually be somewhat active (meaning they're on IRC now and they've been idle <24 hours), @yan12125 and @palinek. IRC like Matrix, Telegram, email, SMS, etc. can be exploited for spam and abuse and so it's important to have admins. IMHO, as many as possible. Problem being that I don't think those ops can add other ops (through ChanServ) as the channel master is agaida. I could get in touch with OFTC about gaining access to the channel if we'd like.

Another thought since the idea of having too few admins is on my mind: what about we create a KeepassX file with the passwords in it and distribute it to team members? In this way everyone will have access to everything in the event of an emergency.

**Reply by yan12125** [reply]
> Another thought since the idea of having too few admins is on my mind: what about we create a KeepassX file with the passwords in it and distribute it to team members? In this way everyone will have access to everything in the event of an emergency.

I had a similiar idea some time ago. It will be great if things goes that way! But that is a quite big and different topic and need a separate discussion thread ;)

**Comment by luis-pereira** [comment]
I don't use Telegram. Never did. Don't have time for it.
My opinion is not an educated one: I'm not against it. That's what I can say.

**Reply by stefonarch** [reply]
I think 2 or better 3 active admins/service is all we need. 
And keeping an eye on "active".

**Reply by telmobss1** [reply]
the telegram is already connected to the matrix, now just wait for the irc

**Reply by yan12125** [reply]
> now just wait for the irc

I have op permission on IRC. Anything I can do?

Just joined the Telegram channel by the way. Thanks for the efforts!

**Reply by paulolieuthier** [reply]
> I have op permission on IRC. Anything I can do?

Are you? When I try to add the IRC integration, I get the message `No chanops found in channel!`, even though you are there.

**Reply by paulolieuthier** [reply]
@agaida can you help here?

**Reply by stefonarch** [reply]
Ehm, agaida is unseen for a year now....

**Comment by GreenLunar** [comment]
Create an official Jabber/XMPP group please.
